### PR TITLE
Tencent Hunyuan Team - Updated Doc for HunyuanDiT

### PR DESCRIPTION
**Pull request by gnobitab**
Hi @yiyixuxu @sayakpaul :

This is the updated doc for HunyuanDiT, please have a look.

Thanks

### CODE REVIEW

File: docs/source/en/api/pipelines/hunyuandit.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+You can find the original codebase at [Tencent/HunyuanDiT](https://github.com/Tencent/HunyuanDiT) and all the available checkpoints at [Tencent-Hunyuan](https://huggingface.co/Tencent-Hunyuan/HunyuanDiT).
+
+**Highlights**: HunyuanDiT supports Chinese/English-to-image, multi-resolution generation.
+
+HunyuanDiT has the following components:
+* It uses a diffusion transformer as the backbone
+* It combines two text encoders, a bilingual CLIP and a multilingual T5 encoder
+
+
```
Comment (sayakpaul) [inline docs/source/en/api/pipelines/hunyuandit.md:31]:
```suggestion

Refer to the [PixArt documentation](https://huggingface.co/docs/diffusers/main/en/api/pipelines/pixart) to know more about how this model can be optimized further. 

```

**Review by sayakpaul — APPROVED** [review]
Looks solid!

File: docs/source/en/api/pipelines/hunyuandit.md (hunk 1 above)
Comment (gnobitab) [reply docs/source/en/api/pipelines/hunyuandit.md:31]: I didn't test it yet so I suggest to defer adding this line.

Comment (sayakpaul) [reply docs/source/en/api/pipelines/hunyuandit.md:31]: No worries. Please disregard in that case.

**Comment by gnobitab** [comment]
updated related files

**Comment by gnobitab** [comment]
@sayakpaul

**Pull request merged**